test/src/server/helpers/revisions.js

```diff
@@ -0,0 +1,415 @@
+import * as error from '../../../../src/common/helpers/error';
+import {
+	createAuthor, createEdition,
+	createEditionGroup,
+	createEditor, createPublisher, createWork,
+	truncateEntities
+} from '../../../test-helpers/create-entities';
+import {
+	getAssociatedEntityRevisions,
+	getOrderedRevisionForEditorPage,
+	getOrderedRevisions
+} from '../../../../src/server/helpers/revisions';
+import chai from 'chai';
+import orm from '../../../bookbrainz-data';
+import {random} from 'faker';
+
+
+const {expect} = chai;
+chai.use(require('chai-sorted'));
+
+
+const {
+	AuthorData, AuthorRevision, Revision, AliasSet,
+	EditionData, EditionRevision, EditionGroupData, EditionGroupRevision,
+	Note, PublisherData, PublisherRevision, WorkData, WorkRevision
+} = orm;
+
+
+describe('getOrderedRevisions', () => {
+	before(async () => {
+		const editor = await createEditor();
+		const editorJSON = editor.toJSON();
+		const revisionAttribs = {
+			authorId: editorJSON.id,
+			id: 1
+		};
+		const promiseArray = [];
+		for (let id = 1; id <= 1000; id++) {
```

Comment:
Umm I made 1000 tests so that the `created_at` column is different in some rows.
I've changed it to 50 revisions and used `fake.date.recent()` to get a recent random date